pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -73,10 +73,16 @@ message ListAndWatchResponse {
 	repeated Device devices = 1;
 }
 
+message TopologyInfo {
+    	int64 socket = 1;
```

Comment:
Are we really talking about a physical socket, or are we talking about a NUMA node?  If we're talking about a physical socket, then we'll need another layer of indirection to determine the NUMA node (since a NUMA node could theoretically be associated with multiple sockets, and one socket can have multiple NUMA nodes).

Reply:
I am in agreement to change the name, if there are no objections from others?

Reply:
I tend to agree in general, but I'm torn on what to do in this specific case.

The terms Socket and NUMA are often thrown around interchangeably since most practical systems typically have a 1-1 mapping between socket and NUMA node. However, as @cbf123 points out, there is actually a many-to-many relationship between them and nothing is preventing you from building a system where a 1-1 mapping does not exist.

Unfortunately, other parts of the K8s code base make the assumption of a 1-1 mapping between socket and NUMA node implicitly. Most notably, the `CPUManager`:

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/topology/topology.go#L33

If this 1-1 mapping assumption is already pervasive throughout the K8s the code base, does it make sense to break that precedent here?

If so, I'd be OK calling this field `node`, so long as we are explicit about it being a NUMANode in its type. I.e.:
```
message NUMANode {
    int64 id = 1;
}
message TopologyInfo {
    	NUMANode node = 1;
}
```

Reply:
Okay I tested and updated to this. If there is agreement on this I will update the Device Manager PR to reflect these changes

Reply:
Works for me.